experiments/collect_unsupported.cpp

```diff
@@ -128,25 +128,34 @@ int main(int argc, char** argv)
    multiset<Feature> unsupported_features;
    using Instance = pair<Feature, size_t>;
    vector<Instance> instances;
-   vector<string> params;
 
    string line;
    getline(input_file, line);
+   unsigned long count=0;
+   cerr << "Scanning " << argv[1] << "...";
    while (true) {
+      ++count;
+      if (count % 1000 == 0) cerr << ".";
       if (line.find("----------At:") == 0) {
+         vector<string> params;
```

Comment:
So each `Feature` was constructed with the parameters of all the previous features? That's unfortunate.

Reply:
Yeah, that's what appeared to be happening :-) params.size() was monotonically increasing...